<launch>
    
    <arg name="arm_1_ns" default="/arm1"/>
    <arg name="arm_2_ns" default="/arm2"/>

    <!-- Dearly Arm 1 -->
    <include file="$(find robot_arm_workcell_manager)/launch/robot_arm_workcell_manager.launch">
        <arg name="arm_ns"          value="$(arg arm_1_ns)"/>
    </include>
  
    <!-- Dearly Arm 2 -->
    <include file="$(find robot_arm_workcell_manager)/launch/robot_arm_workcell_manager.launch">
        <arg name="arm_ns"          value="$(arg arm_2_ns)"/>
    </include>

    <!-- Dearly Arm n.... -->

</launch>
